cdap-watchdog/src/main/java/co/cask/cdap/logging/read/AvroFileReader.java

```diff
@@ -181,9 +174,34 @@ public void readLog(Location file, Filter logFilter, long fromTimeMs, long toTim
     }
   }
 
+  private List<LogEvent> readToNextSync(Filter logFilter, long fromTimeMs, DataFileReader<GenericRecord> dataFileReader,
+                             long startPosition, long endPosition) throws IOException {
```

Comment:
Also, for readability let's rearrange the parameters a bit -
`
private List<LogEvent> readToPrevSync(DataFileReader<GenericRecord> dataFileReader, Filter logFilter, long fromTimeMs, long endPosition)
`